app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("You must be vaccinated!")
+
+        visitor_date = visitor["vaccine"]["expiration_date"]
+        today = date.today()
+        if visitor_date < today:
+            raise OutdatedVaccineError("Your vaccine is outdated!")
+
+        if visitor["wearing_a_mask"] is False:
```

Comment:
Code Efficiency: You can simply use `if not visitor['wearing_a_mask']:` instead of `if visitor['wearing_a_mask'] is False:`. This is more pythonic and readable.